src/main/java/frc/robot/subsystems/ShooterSubsystem.java

```diff
@@ -323,10 +338,10 @@ public double getSpeedRightInRPM() {
 
   // Motor RPM, NOT roller RPM
   private static SpeakerConfig[] speakerConfigList = {
-    new SpeakerConfig(0, 250, 250),
-    new SpeakerConfig(40, 350, 350),
-    new SpeakerConfig(195, 500, 500),
-    new SpeakerConfig(290, 700, 700)
+    new SpeakerConfig(0, 2350, 1950),
+    new SpeakerConfig(40, 2550, 2150),
+    new SpeakerConfig(195, 2750, 2350),
+    new SpeakerConfig(290, 2750, 2350)
   };
 
   private static final SpeakerConfig speakerConfigIdle =
```

Comment:
Since there are constants, its okay to make them `public final`. Then you can use them directly in your JUnit test instead of copy/pasting

Reply:
Resolved in e3171e4